en/Data Structures/Stacks/stack.md

```diff
@@ -1,19 +1,34 @@
 # Stack
 
-Stack is a basic linear data structure which follows an order in which operations are perfomed.The order is called LIFO(Last In First Out)or FILO.
-A perfect example for a stack of plates in a canteen,a pile of books,a box of pringles.
-Stacks are used to implement parsers and evalution expressions and backtracking algorithms. basic operations are pushing an element into stack and popping the element out of the stack.
+A stack is a basic linear data structure that follows an order in which operations are performed. The order is called LIFO(Last In First Out)or FILO.
```

Comment:
I wouldn't call this the "order in which operations are performed"; rather it's the order in which objects can be accessed:

LIFO: The last object to be pushed to the stack is the first object to be popped;
FILO: The first object to be pushed to the stack is the last object to be popped